docs/cassandra.md

```diff
@@ -0,0 +1,64 @@
+| workload.googleapis.com/cassandra.client.request.write.latency.50p              | gauge     | µs          |        | Standard write request latency - 50th percentile |
+| workload.googleapis.com/cassandra.client.request.write.latency.99p              | gauge     | µs          |        | Standard write request latency - 99th percentile |
+| workload.googleapis.com/cassandra.client.request.write.latency.count            | sum       | µs          |        | Total standard write request latency |
+| workload.googleapis.com/cassandra.client.request.write.latency.max              | gauge     | µs          |        | Maximum standard write request latency |
+| workload.googleapis.com/cassandra.client.request.write.timeout.count            | sum       | 1           |        | Number of standard write request timeouts encountered |
+| workload.googleapis.com/cassandra.client.request.write.unavailable.count        | sum       | 1           |        | Number of standard write request unavailable exceptions encountered |
+| workload.googleapis.com/cassandra.client.request.range_slice.latency.50p        | gauge     | µs          |        | Token range read request latency - 50th percentile |
+| workload.googleapis.com/cassandra.client.request.range_slice.latency.99p        | gauge     | µs          |        | Token range read request latency - 99th percentile |
+| workload.googleapis.com/cassandra.client.request.range_slice.latency.count      | sum       | µs          |        | Total token range read request latency |
+| workload.googleapis.com/cassandra.client.request.range_slice.latency.max        | gauge     | µs          |        | Maximum token range read request latency |
+| workload.googleapis.com/cassandra.client.request.range_slice.timeout.count      | sum       | 1           |        | Number of token range read request timeouts encountered |
+| workload.googleapis.com/cassandra.client.request.range_slice.unavailable.count  | sum       | 1           |        | Number of token range read request unavailable exceptions encountered |
+| workload.googleapis.com/cassandra.compaction.tasks.completed                    | sum       | 1           |        | Number of completed compactions since server start |
+| workload.googleapis.com/cassandra.compaction.tasks.pending                      | gauge     | 1           |        | Estimated number of compactions remaining to perform |
+| workload.googleapis.com/cassandra.storage.load.count                            | sum       | bytes       |        | Size of the on disk data size this node manages |
+| workload.googleapis.com/cassandra.storage.total_hints.count                     | sum       | 1           |        | Number of hint messages written to this node since start |
+| workload.googleapis.com/cassandra.storage.total_hints.in_progress.count         | sum       | 1           |        | Number of hints attempting to be sent currently |
```

Comment:
This metric is collected as a Sum/Counter (even in latest release of the jmx-metrics java-contrib component) and is documented as such in [cassandra's docs](https://cassandra.apache.org/doc/latest/cassandra/operating/metrics.html) (search for TotalHintsInProgress) but it really seems like it would be a Gauge. I don't have a system with load on it that is causing this metric to be non-zero but I'm curious if it's correct in the scripts & docs.

Reply:
I took a look into the code.  Cassandra is still using codahale's metric library (Is that dropwizard?).    They're using an underlying counter for this metric in their own code.  I'm not sure the behavior of that library, so this is a good open question.   It's entirely possibly they're sending a monotonic sum expecting users to calculate rate afterwards.

Reply:
What do you mean by "calculate rate afterwards"? The name and description of this metric talk about the number of concurrent hints, which is not a rate of anything (time is not a denominator for that metric).